### ISSUE TITLE
Fail to build serde_macros

### ISSUE THREAD

**Issue by betovsky** [serde-rs/serde#81]
When update Cargo.toml to get serdes_macro, it fails to build:

```
    Updating registry `https://github.com/rust-lang/crates.io-index`
   Compiling aster v0.3.0
     Running `rustc C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\aster-0.3.0\src\lib.rs --crate-name aste
r --crate-type lib -g -C metadata=4062c74cb2dc9294 -C extra-filename=-4062c74cb2dc9294 --out-dir D:\Code\projects\badger
GP\target\debug\deps --emit=dep-info,link -L dependency=D:\Code\projects\badgerGP\target\debug\deps -L dependency=D:\Cod
e\projects\badgerGP\target\debug\deps -Awarnings`
   Compiling quasi v0.3.0
     Running `rustc C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\quasi-0.3.0\src\lib.rs --crate-name quas
i --crate-type lib -g -C metadata=ffe22e38403c370f -C extra-filename=-ffe22e38403c370f --out-dir D:\Code\projects\badger
GP\target\debug\deps --emit=dep-info,link -L dependency=D:\Code\projects\badgerGP\target\debug\deps -L dependency=D:\Cod
e\projects\badgerGP\target\debug\deps -Awarnings`
       Fresh strsim v0.3.0
       Fresh libc v0.1.8
       Fresh regex v0.1.30
       Fresh rustc-serialize v0.3.14
       Fresh rand v0.3.8
       Fresh docopt v0.6.64
       Fresh num v0.1.24
       Fresh serde v0.4.0
C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\quasi-0.3.0\src\lib.rs:112:58: 112:75 error: unresolved name
 `token::NtImplItem`
C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\quasi-0.3.0\src\lib.rs:112         vec![ast::TtToken(self.sp
an, token::Interpolated(token::NtImplItem(self.clone())))]

                        ^~~~~~~~~~~~~~~~~
<std macros>:1:1: 4:58 note: in expansion of vec!
C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\quasi-0.3.0\src\lib.rs:112:9: 112:92 note: expansion site
C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\quasi-0.3.0\src\lib.rs:118:58: 118:76 error: unresolved name
 `token::NtTraitItem`
C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\quasi-0.3.0\src\lib.rs:118         vec![ast::TtToken(self.sp
an, token::Interpolated(token::NtTraitItem(self.clone())))]

                        ^~~~~~~~~~~~~~~~~~
<std macros>:1:1: 4:58 note: in expansion of vec!
C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\quasi-0.3.0\src\lib.rs:118:9: 118:93 note: expansion site
C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\quasi-0.3.0\src\lib.rs:169:57: 169:69 error: unresolved name
 `token::NtArm`
C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\quasi-0.3.0\src\lib.rs:169         vec![ast::TtToken(DUMMY_S
P, token::Interpolated(token::NtArm(self.clone())))]

                       ^~~~~~~~~~~~
<std macros>:1:1: 4:58 note: in expansion of vec!
C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\quasi-0.3.0\src\lib.rs:169:9: 169:86 note: expansion site
error: aborting due to 3 previous errors
Build failed, waiting for other jobs to finish...
C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\aster-0.3.0\src\lib.rs:1:43: 1:66 error: unstable feature
C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\aster-0.3.0\src\lib.rs:1 #![cfg_attr(not(feature = "with-syn
tex"), feature(rustc_private))]

       ^~~~~~~~~~~~~~~~~~~~~~~
note: this feature may not be used in the stable release channel
error: aborting due to previous error
Could not compile `quasi`.

Caused by:
  Process didn't exit successfully: `rustc C:\Users\MB\.cargo\registry\src\github.com-1ecc6299db9ec823\quasi-0.3.0\src\l
ib.rs --crate-name quasi --crate-type lib -g -C metadata=ffe22e38403c370f -C extra-filename=-ffe22e38403c370f --out-dir
D:\Code\projects\badgerGP\target\debug\deps --emit=dep-info,link -L dependency=D:\Code\projects\badgerGP\target\debug\de
ps -L dependency=D:\Code\projects\badgerGP\target\debug\deps -Awarnings` (exit code: 101)
```

**Comment by erickt** [comment]
This should be fixed now in `quasi` v0.3.1! Please reopen if you're still having trouble.

**Comment by betovsky** [comment]
Hi! Is there any plan for the release date of quasi v0.3.1?
